dpp/src/identity/identity.rs

```diff
@@ -152,6 +152,15 @@ impl Identity {
         self
     }
 
+    /// Get the biggest public KeyID
+    pub fn get_public_key_max_id(&self) -> KeyID {
```

Comment:
Good, I needed this change for the bindings. Ended up doing the same thing in the binding directly and do it here later, but you already took care of it :)